src/main/java/org/opentripplanner/inspector/vector/DebugClientAreaStopPropertyMapper.java

```diff
@@ -3,8 +3,8 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.Locale;
-import org.opentripplanner.api.mapping.I18NStringMapper;
-import org.opentripplanner.api.mapping.PropertyMapper;
+import org.opentripplanner.apis.common.mapping.PropertyMapper;
+import org.opentripplanner.ext.restapi.mapping.I18NStringMapper;
```

Comment:
The I18NStringMapper should be moved to `org.opentripplanner.framework.i18n`. No class inside main should have references to Sandbox code (except wiring).

Reply:
8be001d3d0d5f9fe8c234a4ae0ee95f923fc7544